bibtex-actions.el

```diff
@@ -234,12 +234,35 @@ If FORCE-REBUILD-CACHE is t, force reloading the cache."
     (bibtex-actions-refresh force-rebuild-cache))
   bibtex-actions--candidates-cache)
 
+(defun bibtex-actions-complete-key-at-point-org-cite ()
+    "Complete org-cite citation key at point.
+
+When inserting '@' in a buffer the capf UI will present user with
+a list of entries, from which they can narrow against a string
+which includes title, author, etc., and then select one.  This
+function will then return the key 'key', resulting in '@key' at
+point."
+    ;; FIX exit-function is wrong; results in "no match"
```

Comment:
I forgot to remove this comment. This is only true if the below lines are uncommented.